PythonCodeConventions.md

```diff
@@ -5,16 +5,52 @@ PEP8 is the code style guide for Python code used by the main python distributio
 This style guide can be found at:
 https://www.python.org/dev/peps/pep-0008/
 
-Our coding conventions follow this style guide, with the following exceptions:
-* Always spaces. While PEP8 allows tabs for backwards compatibility, we do not.
-* Strings are double quotes. While python allows single and double quoted strings, and PEP8 only says "pick a rule and stick to it". We decided to do double quotes to match C++.
-* "Maximum line length" - PEP8 specifies a maximum line length of 79 characters. At Ultimaker we have no official line length limit. And the general guideline is that the line should fit on your screen.
+Our coding conventions follow the PEP style guide, with the exceptions stated in the generic code_conventions.md provided in this repository. This also means that for rules regarding class naming, documentation, etc., the rules described in the general code convention document MUST be used.
+
+For convenience, these are re-listed here. In case of conflict, the generic code_conventions are considered to be leading.
+* Always use spaces. While PEP8 allows tabs for backwards compatibility, we do not.
+* Strings are double quotes. While Python allows single and double quoted strings, and PEP8 only says "pick a rule and stick to it". We decided to do double quotes to match C++.
```

Comment:
This is double, the part about quotes on strings.

Reply:
Sure, but it gives a bit more info as to why it's not an issue (eg; pep8 sez, whatever, just pick one). 
I'm okay with kicking it out though.